### PR TITLE
Support template

### LINKED ISSUE #42: chore: more issue templates
# What

With #39 and #40, users can now create issues based on template. However, I'd like to have more template, in line to what is done in the [numpy/numpy](https://github.com/numpy/numpy) repository: https://github.com/numpy/numpy/issues/new/choose.

I would love to have templates for
- [ ] Documentation
- [ ] Question/Help/Support

You can get inspiration from the Numpy repository (see above), but I won't be using external links.

# How to

Please, read already existing files in [ISSUE_TEMPLATE](https://github.com/jeertmans/manim-slides/tree/main/.github/ISSUE_TEMPLATE), and get some inspiration from https://github.com/numpy/numpy/issues/new/choose too.

### PR DESCRIPTION
**Pull request by harisaisundhar**
Closes #42 

## Description

-  Template drafted for Question/Help/Support
-  Consists of attributes for criticality of the support and description

## Check List (Check all the applicable boxes)

- [ ] I understand that my contributions needs to pass the checks.
- [ ] If I created new functions / methods, I documented them and add type hints.
- [ ] If I modified already existing code, I updated the documentation accordingly.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2022-10-21 at 12 21 19 PM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Documentation
 description: Ask / Report an issue related to the documentation.
-title: "DOC: <description>"
+title: "DOC: Report an issue"
```
Comment (jeertmans) [inline .github/ISSUE_TEMPLATE/documentation.yml:3]:
```suggestion
title: "DOC: <description>"
```

It is intentional to leave "`<description>`", as it suggests users to replace this with an actual description.

File: .github/ISSUE_TEMPLATE/support.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        </details>
+      options:
+        - "1"
+        - "2"
+        - "3"
+        - "4"
+        - "5"
+    validations:
+      required: true
+
```
Comment (jeertmans) [inline .github/ISSUE_TEMPLATE/support.yml:28]:
```suggestion
```
I think it's the reviewer's job to define how critical an issue is, not the user's.

**Review by jeertmans — CHANGES_REQUESTED** [review]
Hi @harisaisundhar, many thanks for your PR!

I just left two suggested changes, that you can directly apply if you agree.

I prefer to keep the template very minimal for the moment, and later complete it with more information if needed.

Thanks!

**Comment by harisaisundhar** [comment]
Sure! That makes sense. Have merged the changes suggested!

**Comment by jeertmans** [comment]
Thanks @harisaisundhar ! Now merging this

**Pull request merged**